eslint.config.mjs

```diff
@@ -0,0 +1,13 @@
+import checkdigitConfig from '@checkdigit/eslint-config';
+
+export default [
+  ...checkdigitConfig,
+  {
+    rules: {
```

Comment:
Is there a way to turn these rules off only for the stuff inside `date-fns` and `date-fns-tz` directories